plugins/push/frontend/public/javascripts/countly.views.js

```diff
@@ -351,4 +351,13 @@ $(document).ready(function() {
             }
         });
     }
+
+    $('body').on('click', '.tab-buttons > div', function() {
```

Comment:
It would be a good practice to do `.on('click', '.tab-buttons > div').on('click', '.tab-buttons > div', function() {`

Reply:
Also, the selector is too broad, it will work on other cases outside push with the same classes too